pkg/costmodel/allocation.go

```diff
@@ -68,6 +60,11 @@ const (
 	queryFmtLBActiveMins          = `count(kubecost_load_balancer_cost) by (namespace, service_name, %s)[%s:%s]%s`
 )
 
+// This is a bit of a hack to work around garbage data from cadvisor
+// The maximum CPU count on AWS is 96. Standard_M192idms_v2 on azure allows 192. GCP offers 224 in n2ds. Many improvements here are possible to get lower bounnds, but 96 seems reasonable now.
+// Ideally you cap each pod to the max CPU on its node, but that involves a bit more complexity, as it it would need to be done when allocations joins with asset data.
+const MAX_CPU_CAP = 96
```

Comment:
As discussed earlier, I'd bias towards making this ~512+ because we'll still catch the problem we're seeing while also being able to handle more extreme bare metal setups that some users could run.

Reply:
SGTM, will fix.